packages/react-core/src/components/Page/PageSidebar.tsx

```diff
@@ -19,33 +19,49 @@ export interface PageSidebarProps extends React.HTMLProps<HTMLDivElement> {
   theme?: 'dark' | 'light';
 }
 
+export interface PageSidebarContextProps {
+  className: string;
+  nav: React.ReactNode;
+  isNavOpen: boolean;
```

Comment:
are the other ones needed? I think only `isNavOpen` is used

Reply:
Yup, you're right. Wasn't sure if I should leave the others in in case of future stuff, but just isNavOpen is probably better for now.